lib/src/api/method/mod.rs

```diff
@@ -1226,14 +1223,18 @@ where
 		}
 	}
 
+	// TODO: Re-enable doc tests
 	/// Runs a function
 	///
 	/// # Examples
 	///
-	/// ```no_run
+	/// ```ignore
 	/// # #[tokio::main]
 	/// # async fn main() -> surrealdb::Result<()> {
 	/// # let db = surrealdb::engine::any::connect("mem://").await?;
+	/// // Note that the sdk is currently undergoing some changes so the below examples might not
```

Comment:
I'm not sure this note is necessary. Beta does not offer any stability guarantees. By the time we release a stable version, the SDK will be stable at that point.

Reply:
We risk unnecessarily carrying it over to the stable release.

Reply:
The code in the example isn't working right now, if we have a stable release that example needs to be working again it think carrying an example which doesn't work to stable looks worse than having an comment there that it isn't stable.

Reply:
Not working is a bug that needs to be fixed. It shouldn't reach stable like that to begin with. Should we be ignoring the example in the first place?

Reply:
I am ignoring it because it requires a bunch of conversion methods to value which where removed with this PR. I wanted to keep this PR as minimal as possible when it come to conversion with the new value as you mentioned you wanted to discourage people from using value as much as possible. To get this example working we need to add those conversions back which I wanted to leave to a future PR.

Reply:
I have [a PR](https://github.com/surrealdb/surrealdb/pull/4563) open which changes the API to avoid the reliance on `sql::Value` in this PR. In that PR, `sql::Value` is not part of the API at all.